src/routes.tsx

```diff
@@ -54,6 +55,7 @@ setTimeout(() => {
   Success.preload();
   Ref.preload();
   ApiInfo.preload();
+  AuthMod.preload();
```

Comment:
the authMod page can't be reached without a direct link, thus it makes no sense to preload it